doc/implementdoc/param-tech.rst

```diff
@@ -102,7 +102,7 @@ Technical Parameters
     +---------------------------------------------+----+--------------------------------------------+
     |:math:`\overline{K}_{yaf}`                   |MW  |Transmission Capacity Upper Bound           |
     +---------------------------------------------+----+--------------------------------------------+
-    |:math:`T_{af}`                               |MW  |Remaining lifetime of installed transmission|
+    |:math:`T_{af}`                               |MW  |Remaining Lifetime of Installed Transmission|
```

Comment:
Unit 'a'

Reply:
Yes, you are correct. It seems, I have missed this unit mistake.